tox.ini

```diff
@@ -60,7 +60,6 @@ basepython =
     pypy3: pypy3
     flake8,apicheck,linkcheck,configcheck,pydocstyle,bandit: python3.7
     flakeplus: python2.7
-usedevelop = True
```

Comment:
I think this would be necessary for local development and testing. What errors are you observing due to this flag?

Reply:
Hello @georgepsarakis,

> I think this would be necessary for local development and testing.

It's not necessary for local development for sure since it's for `tox` only, and I believe that it's not necessary for local testing because all the tests pass.

> What errors are you observing due to this flag?

You can see a corresponding error on [the latest master build for Python 3.7 on Travis CI](https://travis-ci.org/celery/celery/jobs/569235785).

Reply:
python/typing#573 is probably related to this issue.

Reply:
can we make it False?

Reply:
Sorry, I missed that the `usedevelop` option [was introduced](https://github.com/celery/celery/commit/bd0ed23c81b20fd75c0e2188fcf78e4d74898953) in order to fix coverage measurement.
Let me try to find a better solution.